### ISSUE TITLE
[feature] Python: Ability to import Edit

### ISSUE THREAD

**Issue by enzo-inc** [ast-grep/ast-grep#1186]
# ⭐ Suggestion

It would be nice to be able to import `Edit` from `ast_grep_py` to be used in type annotations.

Current Behaviour:
```
from ast_grep_py import Edit

ImportError: cannot import name 'Edit' from 'ast_grep_py'
```

# 💻 Use Cases

```
from ast_grep_py import SgNode, Edit

def edit_node(node: SgNode) -> Edit: 
...
```

**Comment by HerringtonDarkholme** [comment]
Thanks for reporting! I will release a new version soon!

**Comment by HerringtonDarkholme** [comment]
shipped in ver 0.22.6
